returnn/tf/layers/base.py

```diff
@@ -836,14 +836,18 @@ def add_param(self, param, custom_update=None, trainable=None, saveable=None, ax
       custom_update.set_on_var(param)
     if axes_split_info:
       tf_util.set_param_axes_split_info(param, axes_split_info)
-    if self.reuse_params:
-      name_scope_prefix = self.reuse_params.get_absolute_name_scope_prefix(base_layer=self, param=param)
+    if getattr(param, "RETURNN_layer_map_name", None) is not None:
```

Comment:
It should be `_RETURNN_layer_map_name`.

Reply:
But also, I think you don't need this at all, and this approach only leads to problems.